prov/usnic/src/usdf_ep_rdm.c

```diff
@@ -637,23 +639,31 @@ usdf_ep_rdm_bind_cq(struct usdf_ep *ep, struct usdf_cq *cq, uint64_t flags)
 		return ret;
 	}
 
+	if ((cq->cq_attr.wait_obj == FI_WAIT_FD) ||
+			(cq->cq_attr.wait_obj == FI_WAIT_SET)) {
+		cq->object.fd = eventfd(0, EFD_NONBLOCK | EFD_SEMAPHORE);
```

Comment:
@goodell I agree: I think reading from the fd should reset the value back to 0, because the caller is supposed to drain the queue.